docs/fastdds/dds_layer/core/policy/standardQosPolicies.rst

```diff
@@ -474,24 +483,24 @@ LatencyBudgetQosPolicy
 
 This QoS Policy specifies the maximum acceptable delay from the time the data is
 written until the data is inserted on the DataReader History and notified of the fact.
-That delay by default is set to 0 in order to optimize the internal operations.
+That delay by default is set to 0 in order to optimize the internal operations (see |LatencyBudgetQosPolicy-api|).
 
 List of QoS Policy data members:
 
-+--------------------------+-------------------------------+-------------------+
-| Data Member Name         | Type                          | Default Value     |
-+==========================+===============================+===================+
-| duration                 | fastrtps::Duration_t          | c_TimeZero        |
-+--------------------------+-------------------------------+-------------------+
++----------------------------------------------------------+---------------------------+-------------------------------+
+| Data Member Name                                         | Type                      | Default Value                 |
++==========================================================+===========================+===============================+
+| |LatencyBudgetQosPolicy::duration-api|                   | |Duration_t-api|          | |c_TimeZero-api|              |
```

Comment:
That's because both c_TimeZero and c_TimeInvalid cannot be referenced because sphinx doesn't recognize them. So I have referenced c_TimeInfinite instead so that c_TimeZero and c_TimeInvalid at least have a reference to where they are defined.